deploy/roles/default_role.yaml

```diff
@@ -44,7 +44,7 @@ rules:
       - ''
     resources:
       - configmaps
-    resourceNames: ["assisted-service-leader-election-helper", "assisted-service-migration-helper"]
+    resourceNames: ["assisted-service-leader-election-helper", "assisted-service-migration-helper", "assisted-service-baseiso-helper"]
```

Comment:
we will need gitlab change for this one. And maybe IT ticket to for psi to add new role file